r2dbc-mysql/src/main/java/io/asyncer/r2dbc/mysql/MySqlConnectionConfiguration.java

```diff
@@ -127,6 +128,8 @@ public final class MySqlConnectionConfiguration {
     @Nullable
     private final Publisher<String> passwordPublisher;
 
+    private final AddressResolverGroup<?> resolver;
```

Comment:
This field appears to be @Nullable. Additionally, there is no public method available to set this value. We need to support setting the resolver using Option methods.

You can refer to the following changes to add the new property and create test cases:
https://github.com/asyncer-io/r2dbc-mysql/pull/157

Please feel free to leave a comment if you need any further assistance.